src/lib/operators/index_scan.hpp

```diff
@@ -29,12 +29,8 @@ class IndexScan : public AbstractReadOnlyOperator {
 
   const std::string name() const final;
 
-  /**
```

Comment:
random line.

https://github.com/hyrise/hyrise/blob/e03caf05b0cc48a044ef04593411553ef0013d17/src/lib/operators/index_scan.hpp#L23

This isn't necessary any longer, isn't it?